src/mesh.rs

```diff
@@ -258,44 +177,71 @@ fn load_with_mesh_loader(
     scale: na::Vector3<f32>,
     opt_color: &Option<na::Point3<f32>>,
     group: &mut SceneNode,
+    mut use_texture: bool,
 ) -> Result<SceneNode> {
-    let mesh = mesh_loader::Loader::default()
-        .merge_meshes(true)
-        .load_from_slice(bytes, file_string)
-        .map_err(|e| {
-            if e.kind() == io::ErrorKind::Unsupported {
-                crate::errors::Error::from(format!(
-                    "{file_string} is not supported, because assimp feature is disabled"
-                ))
-            } else {
-                e.into()
+    let mut base = group.add_group();
+    let mut loader = mesh_loader::Loader::default();
+    use_texture &= !is_url(file_string);
+    if use_texture {
+        // TODO: Using fetch_or_read can support remote materials, but loading becomes slow.
```

Comment:
> * ~Use mesh-loader's extendable loader to support remote materials. Previously only remote URDFs and meshes were supported.~ This is disabled for now due to slow. (see also todo comments)

[here](https://github.com/openrr/urdf-viz/tree/taiki-e/remote-material) is a branch that supports remote materials.

You can try how it is slow by checking out that branch and running the following command.

```
cargo run -r -- --disable-assimp https://raw.githubusercontent.com/ToyotaResearchInstitute/hsr_description/HEAD/robots/hsrb4s.obj.urdf --package-path hsr_meshes=https://raw.githubusercontent.com/ToyotaResearchInstitute/hsr_meshes/HEAD
```

Before:
<img width="200" alt="before" src="https://github.com/openrr/urdf-viz/assets/43724913/3bee6897-83d4-4a87-94b1-d43c0918a49c">

After:
<img width="200" alt="after" src="https://github.com/openrr/urdf-viz/assets/43724913/89bb7116-89a2-4f7d-8f27-c9c692da23e1">